app/models/membership.rb

```diff
@@ -25,6 +26,10 @@ def newbie?
     has_post?(DEFAULT_POST_ID) && end_date.nil? && !archived?
   end
 
+  def new_member?
+    has_post?(NEW_MEMBER_ID) && active?
```

Comment:
It seems a little bit weird for me. I mean the user is a new member if it's not newbie? (the `active?` method contain it)